liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -1471,7 +1471,7 @@ private String getSystemServerEnvFilename()
    
 	private List<String> getJPMSOptions() {
 		List<String> args = new ArrayList<String>();
-		File jpmsOptions = new File(containerConfiguration.getWlpHome(), "lib/platform/java/java9.options");
+		File jpmsOptions = new File(containerConfiguration.getWlpHome(), "/lib/platform/java/java9.options");
```

Comment:
I think adding this slash is incorrect as we want the second argument to the `File` constructor to be a relative path.

However, @chyt points out that this method is not currently called, so it's not necessary to fix this.